charts/actions-runner-controller-2/templates/deployment.yaml

```diff
@@ -48,6 +48,9 @@ spec:
         {{- with .Values.imagePullSecrets }}
         - "--auto-scaler-image-pull-secrets={{ include "actions-runner-controller-2.imagePullSecretsNames" . }}"
         {{- end }}
+        {{- if and .Values.manager .Values.manager.flags }}
```

Comment:
do you need to check both?

Reply:
`if and .Values.manager .Values.manager.flags` vs `with .Values.manager.flags.logLevel`

Reply:
```
{{- with .Values.manager.flags.logLevel }}
  - "--log-level={{ . }}"
{{- end }}
```

Reply:
I put it in case a user might delete (intentionally or by mistake) the manager section or the flags section so we don't get an error. If `with` does basically the same thing, I am ok to add `with`

Reply:
Seems to be doing the same thing, updated, thanks!